content/guest/agoldfuss3.md

```diff
@@ -0,0 +1,20 @@
++++
+Title = "Alice Goldfuss"
+date = "2020-10-20T08:32:02-08:00"
+Twitter = "alicegoldfuss"
+Website = "https://blog.alicegoldfuss.com/"
+Type = "guest"
+Facebook = ""
+Linkedin = ""
+Pronouns = ""
+GitHub = ""
+Thumbnail = "img/guests/agoldfuss3.jpg"
+Pinterest = ""
+Instagram = ""
+YouTube = ""
+guest_group = "agoldfuss"
```

Comment:
aww thank you for including this :)

Reply:
(if it's not clear, I'm talking about `guest_group` :)

Reply:
❤️